Parse/Internal/Command/ParseCommand.cs

```diff
@@ -46,14 +46,13 @@ internal class ParseCommand : HttpRequest {
         IList<KeyValuePair<string, string>> headers = null,
         Stream stream = null,
         string contentType = null) {
-      Uri = new Uri(ParseClient.HostName, relativeUri);
+      Uri = new Uri(ParseClient.HostName + relativeUri);
```

Comment:
I swore the issue was that 

    new Uri(new Uri("https://api.parse.com/1/"), "/gogo")

resulted in 

    https://api.parse.com/gogo

Unless I'm reading the [Docs](https://msdn.microsoft.com/en-us/library/9hst1w91(v=vs.110).aspx) wrong...

As long as all of our relativeUris don't have leading slashes (which I thought I got rid of in #77), we should keep the path part of the `HostName`.